File: AtCoder/abc466_A.cpp
#include<iostream>
using namespace std;

int main(){
    int n;
    cin >> n;

    bool flag = 1;

    int num[100];

    for(int i=0; i<n; i++){
        int m;
        cin >> m;
        num[i] = m;
    }

    for(int i=0; i<n; i++){
        if(num[i] >= 0){
            flag = 0;
            break;
        }
    }

    if(flag == 0){
        cout << "No\n";
    }
    else{
        cout << "Yes\n";
    }
}

File: AtCoder/abc466_A.py
n = int(input())
num = list(map(int, input().split(" ")))
flag = 1

for i in range(len(num)):
    if num[i] >= 0:
        flag = 0
        break

if flag == 0:
    print("No")
else:
    print("Yes")
